e2e/docs/.vuepress/config.ts

```diff
@@ -69,6 +69,7 @@ export default defineUserConfig({
           ],
         },
       ],
+      '/heading-sidebar/': 'heading',
```

Comment:
Could you change the strcture to:

```
sidebar/
  auto/
  auto.md
  config/
    1.md
    2.md
  README.md
```

I believe this could be better, thanks.

Reply:
To clarify, are you suggesting we remove the `heading-sidebar` folder and replace the `sidebar` folder with this structure?

![Screenshot 2024-01-28 at 9 13 51 AM](https://github.com/vuepress/ecosystem/assets/30451189/f1c0fd7f-bffc-4724-b5da-66ec1c4d07e4)

Reply:
Yes! but a auto folder and a auto.md, auto folder is for theme config, and auto.md is for frontmatter.